flag/flag.go

```diff
@@ -35,7 +35,10 @@ func Init(f interface{}) {
 }
 
 func Parse(v *viper.Viper, fs *pflag.FlagSet) {
-	v.BindPFlags(fs)
+	err := v.BindPFlags(fs)
+	if err != nil {
+		panic(err)
```

Comment:
Rest of the code follows this pattern for the error handling. Will test in release-operator before merging.